core/src/test/java/hudson/logging/LogRecorderTest.java

```diff
@@ -68,7 +68,7 @@
         assertEquals(lr.handler.getView().get(0), record);
         assertEquals(1, lr.handler.getView().size());
 
-        lr.doClear();
+        lr.handler.clear();
```

Comment:
this calls the handler code rather than the jelly method that now has a permission check, otherwise it get's instance or null exceptions, (none of the other code in this test calls jelly code that hit's permission checks)

another approach would be to change the production code to something like:
```
Jenkins jenkins = Jenkins.getInstanceOrNull();
if (jenkins != null) {
   jenkins.checkPermission(Jenkins.ADMINISTER);
}
```

or remove the test

or introduce the jenkins rule...

Reply:
or mock it.